server/src/scoring/validations/userValidate.ts

```diff
@@ -0,0 +1,11 @@
+﻿import { User } from '@prisma/client';
+import Joi from 'joi';
+
+const schema = Joi.object().keys({
+  nickname: Joi.string().allow(null, ''),
+  contractType: Joi.string().required(),
+  toWorkDistance: Joi.number().required(),
+});
+export const validateUser = (data: User) => {
```

Comment:
Dane które przyjmuje ta funkcja na pewno nie są typu user, ja widziałbym tu zastosowanie dla `unknown` a jeżeli joi nie będzie chciał przyjąć to any

Reply:
ok